privacy/anonymizing-your-mac-address.md

```diff
@@ -32,6 +32,10 @@ $ sudo NetworkManager -V
 
 Right click on the Network Manager icon of your NetVM in the tray and click 'Edit Connections..'.
 
+## Notes
+
+If you choose to create a new NetVM instead of modifying sys-net you will want to follow the instructions on [Installing and updating software in VMs](https://www.qubes-os.org/doc/software-update-vm/) to tell your Template VMs which NetVM to use.
+
```

Comment:
This looks like it confusingly breaks up a series of instructions.

Reply:
Maybe not using a heading here is the way to go. You could just do something like:

```
**Note:** If you choose to [...]
```